docs/src/main/tut/datatypes/validated.md

```diff
@@ -256,47 +259,97 @@ For example:
 
 ```tut:book
 FormValidatorNel.validateForm(
-            username = "Joe",
-            password = "Passw0r$1234",
-            firstName = "John",
-            lastName = "Doe",
-            age = 21
-          )
+  username = "Joe",
+  password = "Passw0r$1234",
+  firstName = "John",
+  lastName = "Doe",
+  age = 21
+)
 
 FormValidatorNel.validateForm(
-            username = "Joe%%%",
-            password = "password",
-            firstName = "John",
-            lastName = "Doe",
-            age = 21
-          )
+  username = "Joe%%%",
+  password = "password",
+  firstName = "John",
+  lastName = "Doe",
+  age = 21
+)
 ```
 
 Sweet success! Now you can take your validation process to the next level!
 
-### Coming from `Either`-based validation
+### A short detour
+
+Typically, you'll see that `Validated` will be accompanied by a `NonEmptyList` when it comes to accumulation. The thing here is that you can define your own accumulative data structure and you're not limited to the aforementioned construction.
+
+For doing this, you have to provide a `Semigroup` instance. `NonEmptyList`, by definition has its own `Semigroup`. For those who don't know what a `Semigroup` is, let's see a simple example.
+
+#### Accumulative Structures
+
+According to [Wikipedia](https://en.wikipedia.org/wiki/Semigroup):
```

Comment:
Instead of linking to wikipedia, I think the link to cats documentation should be enough :) 
Maybe we could just say " For those who don't know what a `Semigroup` is, you can find out more [here](../typeclasses/semigroup.html)."

What do you think? :)

Reply:
Sounds good! I'll fix it.